boards/shields/waveshare_rs485_can/doc/index.rst

```diff
@@ -0,0 +1,122 @@
+.. _waveshare_rs485_can:
+
+WAVESHARE-RS485-CAN: RS485 CAN expansion board
+##############################################
+
+Overview
+********
+The RS485 CAN Shield will easily enable RS485/CAN communication functions for
+NUCLEO/XNUCLEO Arduino boards.
```

Comment:
I don't think you should refer to nucleo boards but rather remain vendor neutral.

Reply:
Much better, thanks.